mjpeg-view/src/main/java/com/github/niqdev/mjpeg/MjpegInputStreamDefault.java

```diff
@@ -22,7 +22,7 @@
     private final static int FRAME_MAX_LENGTH = 40000 + HEADER_MAX_LENGTH;
     private final byte[] SOI_MARKER = {(byte) 0xFF, (byte) 0xD8};
     private final byte[] EOF_MARKER = {(byte) 0xFF, (byte) 0xD9};
-    private final String CONTENT_LENGTH = "Content-Length";
+    private final String CONTENT_LENGTH = "Content-length";
```

Comment:
This is redundant 
<img width="768" alt="image" src="https://user-images.githubusercontent.com/3314607/142197357-efe3aaeb-588f-407b-8af2-06fd8503eade.png">